examples/provisioner/multiple-ebs.yaml

```
@@ -0,0 +1,46 @@
+# This example provisioner will provision instances
+# with multiple EBS attached
+
+apiVersion: karpenter.sh/v1alpha5
+kind: Provisioner
+metadata:
+  name: default
+spec:
+  requirements:
+    # Include general purpose instance families
+    - key: karpenter.k8s.aws/instance-family
+      operator: In
+      values: [c6g, c7g, c6a, c6i, m6a, m6g, m6i, r6a, r6g, r6i]
+    # Exclude small instance sizes
+    - key: karpenter.k8s.aws/instance-size
+      operator: In
+      values: [medium, large, xlarge, xlarge, 2xlarge, 4xlarge]
+  providerRef:
+    name: my-provider
+---
+apiVersion: karpenter.k8s.aws/v1alpha1
+kind: AWSNodeTemplate
+metadata:
+  name: my-provider
+spec:
+  subnetSelector:
+    karpenter.sh/discovery: "${CLUSTER_NAME}" # replace with your cluster name
+  securityGroupSelector:
+    karpenter.sh/discovery: "${CLUSTER_NAME}" # replace with your cluster name
+  amiFamily: Ubuntu
```

I believe ubuntu uses `/dev/sda` device names.

```suggestion
  amiFamily: AL2
```

---

You are right. Been a while sinew I did partitions on Ubuntu.
I'm offline for today. Will try to update tomorrow.

---

```suggestion
  amiFamily: Bottlerocket
```
?

---

That's the other one.